cypress/page/products-list.page.ts

```diff
@@ -4,9 +4,8 @@ class ProductsListPage {
 
   constructor() {
     this.addToCartBtn =
-      "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
-    this.proceedToCheckoutBtn =
-      "[style*='display: block;'] .button-container > a";
+      ".product-container .button-container > a.ajax_add_to_cart_button";
```

Comment:
see if you can use an attribute to indicate that we are looking for the "Add to cart" button